app/assets/stylesheets/active_admin/structure/_main_structure.scss

```diff
@@ -1,29 +1,23 @@
-#wrapper {
-  width: 100%;
-}
-
-.index #wrapper {
-  display: table;
-}
-
 #active_admin_content {
-  margin: 0;
+  @include clearfix;
   padding: $horizontal-page-margin;
+}
 
-  #main_content_wrapper {
-    float: left;
-    width: 100%;
+#main_content_wrapper {
```

Comment:
Avoid using id selectors<br>Selector `main_content_wrapper` should be written in lowercase with hyphens